lib/components/user/monitored-trip/trip-monitored-days.tsx

```diff
@@ -0,0 +1,81 @@
+import { FormattedList, FormattedMessage, useIntl } from 'react-intl'
+
+import InvisibleA11yLabel from '../../util/invisible-a11y-label'
+import React from 'react'
+
+import { ALL_DAYS } from '../../../util/monitored-trip'
+import { getBaseColor } from '../../util/base-color'
+import FormattedDayOfWeek from '../../util/formatted-day-of-week'
+import FormattedDayOfWeekCompact from '../../util/formatted-day-of-week-compact'
+
+import styled from 'styled-components'
+
+interface Props {
+  days: string[]
+}
+
+const DayCircleContainer = styled.div`
+  display: flex;
+  gap: 4px;
+`
+
+const MonitoredDayCircle = styled.span<{
+  baseColor: string
+  monitored: boolean
+}>`
+  align-items: center;
+  background-color: ${(props) =>
+    props.monitored ? props.baseColor : 'transparent'};
+  border-radius: 50%;
+  color: ${(props) => (props.monitored ? 'white' : 'inherit')};
+  display: flex;
+  height: 3rem;
+  justify-content: center;
+  opacity: ${(props) => (props.monitored ? 1 : 0.7)};
+  text-transform: capitalize;
+  width: 3rem;
+
+  span {
```

Comment:
When setting this to a font with a better bounding box things center better!
<img width="66" alt="Screenshot 2024-04-19 at 11 36 07 AM" src="https://github.com/opentripplanner/otp-react-redux/assets/86619099/3b267957-3ff0-492a-b10a-edbe838589ee">

Reply:
This is `SF Pro Display`. Not sure what we should do for windows. https://tonsky.me/blog/centering/

Reply:
`Inter`?

Reply:
Monospace font :/